plugins/rapid7_insightidr/help.md

```diff
@@ -1673,6 +1673,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 4.1.1 - Fix issue with endpoint agent enum 
```

Comment:
Can you please include the action name and also more details about the issue that is fixed?

Reply:
```suggestion
* 4.1.1 - Advanced Query on Log Set Action: Updated  EndPoint Agent to Endpoint Agent in log_set
```